pyvista/core/utilities/geometric_sources.py

```diff
@@ -3682,9 +3688,8 @@ def _make_default_part(geometry: str) -> PolyData:
                 must_contain=geometry,
                 name='Geometry',
             )
-            raise NotImplementedError(
-                f"Geometry '{geometry}' is not implemented"
-            )  # pragma: no cover
+            msg = f"Geometry '{geometry}' is not implemented"
+            raise NotImplementedError(msg)  # pragma: no cover
```

Comment:
Some of the unreported coverage was previously ignored but now since it's split across multiple lines we have one line skipped and the other is not skipped. I've seen this in a few spots, I think we should address this. Either add the skip to both lines, or maybe move the comment up to the closest branch (e.g. one line above, if applicable)
```suggestion
            msg = f"Geometry '{geometry}' is not implemented"  # pragma: no cover
            raise NotImplementedError(msg)  # pragma: no cover
```